labs/instana-v2/opentelemetry/2-lab-environment/index.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 2. Lab Environment
+description: Instana lab environment setup
+sidebar_position: 2
+---
+
+# Lab Environment
+
+The lab environment includes Openshift cluster with Instana.
+
+Openshift clusters:
+
+1. **Instana cluster** - An instance of Single Node OpenShift cluster that will
+   be used to install the Instana agent. The OpenTelemetry Demo Application and Instana backend will be pre-installed
+
+:::note
+
+- IBM Instana cluster have been pre-installed.
+- OpenTelemetry Demo Application have been pre-installed.
+- You will adjust configuration of the OpenTelemetry Demo Application and enable OpenTelemetry for Instana agent during the lab.
+
+:::
+
+![](images/architecture.png)
+
+## Prerequisites
+
+To complete this lab you will need:
+
+- Instana Download Key - Required when requesting lab environment
```

Comment:
Can you please update this to the below for when we get the demo environment ready:

- Instana License
    - Agent Key - Entered when requesting lab environment
    - Sales Key - Entered when requesting lab environment